src/DssSpell.t.sol

```diff
@@ -502,97 +536,98 @@ contract DssSpellTest is DSTest, DSMath {
 
         // box values in RAD
         {
-            uint normalizedBox = values.cat_box * RAD;
+            uint256 normalizedBox = values.cat_box * RAD;
             assertEq(cat.box(), normalizedBox);
         }
 
         // check number of ilks
         assertEq(reg.count(), values.ilk_count);
-
-        // check OsmMom authority
-        assertEq(osmMom.authority(), values.osm_mom_authority);
-
-        // check FlipperMom authority
-        assertEq(flipMom.authority(), values.flipper_mom_authority);
```

Comment:
Same comment as above, we have a more complete system test if we leave these.

Reply:
Added them back.